README.md

```diff
@@ -205,19 +205,16 @@ configuration.securityProtocol = .plaintext
 
 ```swift
 let leafCert = KafkaConfiguration.TLSConfiguration.LeafAndIntermediates.pem("YOUR_LEAF_CERTIFICATE")
-let rootCert = KafkaConfiguration.TLSConfiguration.Root.pem("YOUR_ROOT_CERTIFICATE")
+let trustRoots = KafkaConfiguration.TLSConfiguration.TrustRoots.pem("YOUR_ROOT_CERTIFICATE")
 
 let privateKey = KafkaConfiguration.TLSConfiguration.PrivateKey(
     location: .file(location: "KEY_FILE"),
     password: ""
 )
 
-let tlsConfig = KafkaConfiguration.TLSConfiguration.keyPair(
-    privateKey: privateKey,
-    publicKeyCertificate: leafCert,
-    caCertificate: rootCert,
-    crlLocation: nil
-)
+var tlsConfig = KafkaConfiguration.TLSConfiguration()
+tlsConfig.clientIdentity = .none
+tlsConfig.brokerVerification = .verify(certificateRevocationListPath: nil)
```

Comment:
We are not longer using the above defined certs and key which seems unintentionally.

Reply:
Thanks for pointing that out! Reduced the security examples to a minimum now!